app/views/boards/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Boards</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @boards.each do |board| %>
+      <tr>
```

Comment:
Can we use a `partial` for this boards? So Rails can optimize the renders if you [render a collection](https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections). You can see [here](https://medium.com/@coorasse/partial-rendering-performance-in-rails-101fdfb6ffb9) a post about this performance I mention